dm-csc-base/meta.yaml

```diff
@@ -0,0 +1,24 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: dm-csc-base
+  version: 1.0.0.rc5
+
+source:
+  git_url: https://github.com/lsst-dm/dm_csc_base
+  git_rev: master
```

Comment:
This is built using git master branch. Wouldn't it be better to use tagged tar.gz file? or maybe this file is used for development purposes.

Reply:
I switched this to tags/1.0.0-rc6